components/messages/HowTo.module.css

```diff
@@ -0,0 +1,27 @@
+.howtocontainer {
+  height: 100%;
+  display: grid;
+  grid-template-rows: repeat(6, 1fr);
+  background-color: var(--main-color);
+}
+
+.title {
+  font-family: Ranchers, sans-serif;
+  color: var(--primary-font-color);
+  text-align: center;
+  font-size: 1.8rem;
+  align-self: center;
+}
+
+.info {
+  font-family: Roboto, sans-serif;
+  color: var(--primary-font-color);
+  text-align: start;
+  padding-left: 1.5rem;
+  font-size: 1rem;
+  align-self: center;
+}
+
+.gsbtn {
```

Comment:
Naming could be more descriptive. As somebody who doesn't know the project it's hard to figure out what 'gs' stands for. You could name it `.startBtn` i.e.